File: Training/OOPCorePlatform/C#/Exercises/before/4 Collections/Person.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace _4_Collections
{
    class Person
    {
        private string _firstName;
        private string _lastName;
        private string _ssn;

        public Person(string firstName, string lastName, string ssn)
        {
            _firstName = firstName;
            _lastName = lastName;
            _ssn = ssn;
        }

        public string FirstName { get
            {
                return _firstName;
            }
        }
        public string LastName { get
            {
                return _lastName;
            }
        }
        public string Ssn {get
            {
                return _ssn;
            }
        }


        // TODO: Create an  Equals method that takes a Person as a parameter and do a comparison using the ssn field
       
       

        public override string ToString()
        {
            return string.Format("{0}, {1} ({2})", _lastName, _firstName, _ssn);
        }
    }
}

File: Training/OOPCorePlatform/Java/Exercises/My Answers/02 Formatting/src/Person.java

public class Person {

	private String firstName;
	private String lastName;
	
	public Person(String firstName, String lastName) {
		this.firstName = firstName;
		this.lastName = lastName;
	}
	public String getFirstName() {
		return this.firstName;
	}
	public String getLastName() {
		return this.lastName;
	}
	@Override
	public String toString() {
		return String.format("%s, %s", this.lastName, this.firstName);
	}
}
